dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/FileVerificationService.kt

```diff
@@ -8,40 +8,40 @@ import org.springframework.web.multipart.MultipartFile
 import java.io.IOException
 
 /**
- * A service for performing basic PDF sanity checks
+ * A service for performing basic sanity checks on files uploaded by user
  */
 @Component
-class PdfVerificationService {
+class FileVerificationService {
     private val logger = LoggerFactory.getLogger(javaClass)
 
     /**
-     * A function that performs surface-level checks to ensure that an uploaded document is indeed a PDF.
+     * A function that performs surface-level checks to ensure that an uploaded file is indeed a PDF.
      * This function does not enforce anything else.
      * In particular a file passing this function is in no way guaranteed to be safe to open.
      */
-    fun assertThatDocumentLooksLikeAPdf(document: MultipartFile, correlationId: String) {
+    fun assertThatFileLooksLikeAPdf(file: MultipartFile, correlationId: String) {
```

Comment:
It seems like that at first sight.
But in reality, it does two checks:
1) It checks (tries) if it can be parsed as pdf
2) If it can be parsed as pdf, it is checked if its is empty

Reply:
Maybe name it "assertThatFileIsParsableAsPdfWithAtLeastOnePage" ?

Reply:
I'm fine then as is. 👍